crates/codegen/parser/runtime/src/visitor.rs

```diff
@@ -2,45 +2,33 @@ use std::rc::Rc;
 
 use super::{cst::*, cursor::Cursor};
 
-#[allow(unused_variables)]
 pub trait Visitor<E> {
-    fn rule_enter(
-        &mut self,
-        node: &Rc<RuleNode>,
-        cursor: &Cursor,
-    ) -> Result<VisitorEntryResponse, E> {
+    fn rule_enter(&mut self, _: &Rc<RuleNode>, _: &Cursor) -> Result<VisitorEntryResponse, E> {
```

Comment:
I think `_node` etc is preferable because it documents the role of the arg.

Reply:
The type pretty much documents what it is in this case, but I can also use the prefixed name, if preferred

Reply:
The type documents it only with some reasoning applied, whereas the name is immediate.

Reply:
Done in d5e851da266782c1659ed119c043ab5de06e2486